uo_sys_framework/ShotCountModule/cfgFunctions/common/fn_shotCountInit.sqf

```diff
@@ -1,37 +1,40 @@
-/*	Description: Gives players custom self actions 
+/*	Description: Gives players custom self actions
  *	Arguments:
  * 		N/A
  *	Return Value:
- * 		ARRAY 	
+ * 		ARRAY
  *	Author
  *		Beta & TinfoilHate & PiZZADOX
  */
 
+#define COMPONENT ShotCount
 #include "\x\UO_FW\addons\main\script_macros.hpp"
+UO_FW_EXEC_CHECK(ALL);
 
+if !(UO_FW_SERVER_SHOTCOUNTMODULE_ALLOWED) exitwith {};
 
 if (!(missionNamespace getVariable ["UO_FW_ShotCount_Enabled",false])) exitwith {};
-["Shot Count", "Count shots fired by units", "Sacher"] call UO_FW_FNC_RegisterModule; 
+["Shot Count", "Count shots fired by units", "Beta and TinfoilHate and PiZZADOX and Sacher"] call UO_FW_FNC_RegisterModule;
 if (isServer) then
 {
 	aCount_west_ExpendedAmmunition = [];
 	aCount_resistance_ExpendedAmmunition = [];
 	aCount_east_ExpendedAmmunition = [];
 	aCount_civilian_ExpendedAmmunition = [];
 	aCount_classNames = [];
-	
+
 	[{ time > 0.1 }, {
```

Comment:
CBA_missionTIme > 0.1 or if this is such a short delay, you can use execNextFrame which causes less overhead

Reply:
Hmm looking at this I wonder if XEH is a better way of doing this compared to doing an `allMissionObjects` foreach/apply.

Reply:
> 
> 
> Hmm looking at this I wonder if XEH is a better way of doing this compared to doing an `allMissionObjects` foreach/apply.

That would be a much better method.

Reply:
XEH initpost method implemented